dashboard/app/models/user.rb

```diff
@@ -2570,6 +2571,11 @@ def dependent_students
     dependent_students
   end
 
+  def dependent_students_count
+    # Limit the number of students to mitigiate slow page loads on /users/edit
+    students.limit(Policies::User::DEPENDENT_STUDENTS_COUNT_LIMIT).uniq.count
```

Comment:
I'm gonna test and stress this query out myself, but I have one question: is it that big of a deal if this returns a value less than the `DEPENDENT_STUDENTS_COUNT_LIMIT`? I think the subquery would return at most that limit, but since there may be a few duplicates, it would return a count less than that after the `uniq` or does the `uniq` qualify the subquery as `SELECT DISTINCT`? I think this is just ActiveRecord trivia, but I'm curious.

Reply:
Oh, that's a good question. If we never reach `DEPENDENT_STUDENTS_COUNT_LIMIT` due to `uniq` trimming some of the returned students, the whole logic around communicating the overage (using `+` or `over`) is pointless, as we'd never reach the case where `dependentStudentCount === dependentStudentCountLimit`.